Apps/Src/IOState.c

```diff
@@ -0,0 +1,93 @@
+#include "common.h"
+#include "os_cfg_app.h"
+#include "IOState.h"
+#include "Tasks.h"
+#include "Pedals.h"
+#include "Ignition.h"
+#include "Dashboard.h"
+#include "StatusLeds.h"
+
+static void putIOState(void);
+
+#define IO_STATE_DLY_MS 250u 
+#define IO_STATE_HEARTBEAT_DELAY (1000u)/IO_STATE_DLY_MS
+
+void putIOState(void){
+    CANDATA_t message;
+
+    // Clear message before sending
+    memset(&message, 0, sizeof message);
+    message.ID = IO_STATE;
+
+    // Get pedal information
+    message.data[0] = Pedals_Read(ACCELERATOR);
+    int8_t brake_pedal = Pedals_Read(BRAKE);
+    message.data[1] = brake_pedal;
```

Comment:
since brake_pedal returns int8_t but the can payload expects a uint_8. it really shouldn't matter cuz we don't return a negative number from Pedals_Read:
```suggestion
    message.data[1] =  (uint8_t)brake_pedal;
```

Reply:
@etejeda113 is working on this bug #443 . should be merged in before wednesday.

Reply:
Pedals_Read() is bounded from 0-100 alr tho right?

Reply:
yes